BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -237,6 +240,10 @@ fileprivate final class ScrollerWrapperView: UIView {
         if self.scrollView.showsHorizontalScrollIndicator != scrollView.showsHorizontalScrollIndicator {
             self.scrollView.showsHorizontalScrollIndicator = scrollView.showsHorizontalScrollIndicator
         }
+        
+        if self.scrollView.keyboardDismissMode != scrollView.keyboardDismissMode {
```

Comment:
Why is this conditional needed?

Reply:
Ah right, I need to get rid of this since your PR added this in.

As for the conditional, I _think_ this is a safety net to ensure the scroll view isn’t resetting any of its own state if the value didn’t change (even though in theory, it shouldn’t) - the other properties within this apply method follow the same convention.

Reply:
I probably wrote most of the other conditionals – `UIScrollView` has a few properties (including `frame`, `contentSize`, and insets, IIRC) that reset any in-progress scrolling when they are set, regardless of whether the values have actually changed.